src/routes/results.js

```diff
@@ -1,3 +1,68 @@
+const findAll = require('../find_all.js');
+
 module.exports = (req, res) => {
-  res.render('search_results');
+  let options = {};
+  const filtersArray = [
+    req.query.search ? 'search-' : 'null-',
+    req.query.stage ? 'stage-' : 'null-',
+    req.query.sector ? 'sector' : 'null'
+  ].join('')
+
+  switch (filtersArray) {
+    case 'search-stage-sector':
```

Comment:
maybe change the null to 'null-the filter missing' for example in your second case 'search-stage-noSector'